<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="GOLDHOUSE-PC" name="Default test" tests="1" failures="1" timestamp="7 Feb 2017 13:13:24 GMT" time="59.183" errors="0">
  <testcase name="helpcentralTitleTest" time="0.803" classname="com.yahoo.ycem.tests.helpcentral.HelpCentralTest">
    <failure type="java.lang.RuntimeException" message="Failed to open the url on browser: java.lang.NullPointerException: null value in entry: url=null">
      <![CDATA[java.lang.RuntimeException: Failed to open the url on browser: java.lang.NullPointerException: null value in entry: url=null
	at com.yahoo.ycem.tests.pages.BasePage.load(BasePage.java:86)
	at com.yahoo.ycem.tests.pages.BasePage.<init>(BasePage.java:46)
	at com.yahoo.ycem.tests.pages.helpcentral.HelpCentralPage.<init>(HelpCentralPage.java:108)
	at com.yahoo.ycem.tests.helpcentral.HelpCentralTest.helpcentralTitleTest(HelpCentralTest.java:33)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:84)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:714)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:335)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:330)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
	at org.testng.SuiteRunner.run(SuiteRunner.java:240)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1224)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1149)
	at org.testng.TestNG.run(TestNG.java:1057)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:111)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:204)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:175)
]]>
    </failure>
  </testcase> <!-- helpcentralTitleTest -->
</testsuite> <!-- Default test -->
